compiler/modules/types/list.bal

```diff
@@ -241,17 +244,17 @@ function listInhabited(Context cx, FixedLengthArray members, SemType rest, Conju
         int negLen = nt.members.fixedLength;
         if len < negLen {
```

Comment:
I am suspicious that `negLen` is left unchanged here but is changed below in the `foreach`. Are you sure that's right?